### PR TITLE
Validate Regex Input in Regex Blacklist

### PR DESCRIPTION
**Pull request by bgalek**

### CODE REVIEW

**Comment by Mcat12** [comment]
@bgalek please review the last bullet point of the getting started guide: https://github.com/pi-hole/web#getting-started-development (base off of `development` and merge into that branch). Also make sure to sign-off your commits: https://github.com/pi-hole/web/pull/94/checks?check_run_id=25022724

It would probably be best to wait until #92 is merged, so the regex validator can fit in that framework.

**Comment by bgalek** [comment]
@Mcat12 branch updated, pr changed to development ;) If you notify me when #92 is merged, I can update this branch if needed ;)

**Comment by Mcat12** [comment]
:+1: You can also subscribe to that PR and GitHub might also update you.

**Comment by Mcat12** [comment]
@bgalek #92 is merged

File: src/validate.test.js [hunk 1]
```diff
@@ -118,4 +118,25 @@ describe("Testing the validation functions", () => {
...
   });
+
+  describe("isValidRegex", () => {
+    it("should validate actual regex", () => {
+      const SIMPLE_REGULAR_EXPRESSION = "[0-9]+";
+      const COMPLX_REGULAR_EXPRESSION =
+        "^([0-9a-zA-Z]([-.\\w]*[0-9a-zA-Z])*@([0-9a-zA-Z][-\\w]*[0-9a-zA-Z]\\.)+[a-zA-Z]{2,9})$";
+
+      expect(isValidRegex(SIMPLE_REGULAR_EXPRESSION)).toBe(true);
+      expect(isValidRegex(COMPLX_REGULAR_EXPRESSION)).toBe(true);
```
Comment (Mcat12) [inline src/validate.test.js]: Only test one regex in each test function.

**Comment by Mcat12** [comment]
Closes #85

**Pull request merged**